packages/metrics-discovery/src/metrics-discovery.ts

```diff
@@ -0,0 +1,53 @@
+import type { NodeStructOutput } from '@river-build/generated/dev/typings/INodeRegistry'
+import { RiverRegistry, getWeb3Deployment } from '@river-build/web3'
+import { ethers } from 'ethers'
+
+const NODE_STATUS = {
```

Comment:
Can we not read the node_status enum from abi file in web3 instead of copying it here with ellisions ?

Reply:
@jterzis that was my original intention too, but i couldn't find them. do you know where they are?